src/plugin-types.ts

```diff
@@ -99,7 +107,7 @@ export interface TraceAgent {
    * null as an argument.
    * @returns The return value of calling fn.
    */
-  runInRootSpan<T>(options: RootSpanOptions, fn: (span: SpanData|null) => T): T;
+  runInRootSpan<T>(options: RootSpanOptions, fn: (span: SpanData) => T): T;
```

Comment:
I am pretty sure that this type change is not semver major. Did you think about this as well, and agree?

Reply:
I guess the semver-major change is that if someone had significant logic branching based on the root span being null, then that code path will no longer execute. I don't expect users of the custom tracing API to encounter this at all, but I did have to change every plugin.